redis-utils/src/main/java/org/ballerinalang/redis/actions/StringActions.java

```diff
@@ -0,0 +1,459 @@
+     * @param key key key
+     * @param redisValue value
+     * @return `OK` if successful
+     */
+    public static Object set(HandleValue redisDataSourceHandleValue, String key, String redisValue) {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return set(key, redisValue, redisDataSource);
+        } catch (Throwable e) {
+            return BallerinaErrors.createError(REDIS_EXCEPTION_OCCURRED, e.getMessage());
+        }
+    }
+
+    /**
+     * Sets or clears the bit at offset in the string value stored at key.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key key
+     * @param value value
+     * @param offset The offset at which the value should be set
+     * @return The original bit value stored at offset
+     */
+    public static Object setBit(HandleValue redisDataSourceHandleValue, String key, int value, int offset)  {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return setBit(key, value, offset, redisDataSource).intValue();
+        } catch (Throwable e) {
+            return BallerinaErrors.createError(REDIS_EXCEPTION_OCCURRED, e.getMessage());
+        }
+    }
+
+    /**
+     * Set the value and expiration of a key.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key key
+     * @param value value
+     * @param expirationPeriodSeconds Expiration time to be set, in seconds
+     * @return New value of the key or
+     */
+    public static Object setEx(HandleValue redisDataSourceHandleValue, String key, String value,
+                               int expirationPeriodSeconds)  {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return setEx(key, value, expirationPeriodSeconds, redisDataSource);
+        } catch (Throwable e) {
+            return BallerinaErrors.createError(REDIS_EXCEPTION_OCCURRED, e.getMessage());
+        }
+    }
+
+    /**
+     * Set the value of a key, only if the key does not exist.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key key
+     * @param value value
+     * @return New value of the key
+     */
+    public static Object setNx(HandleValue redisDataSourceHandleValue, String key, String value)  {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return setNx(key, value, redisDataSource).booleanValue();
+        } catch (Throwable e) {
+            return BallerinaErrors.createError(REDIS_EXCEPTION_OCCURRED, e.getMessage());
+        }
+    }
+
+    /**
+     * Overwrite part of a string at key starting at the specified offset.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key key
+     * @param offset
```

Comment:
Can you add the description? Check all places as well.

Reply:
Updated in e26d6d2